### DISCUSSION TITLE
[Reorganize docs of functions]

### DISCUSSION THREAD

**Discussion by jtlap** [jfalcou/eve#1451] [Research & Development]
After adding new complex implementations of many math functions, I wonder if the doc structure  of functions could not be changed in three  parts :

1) functions that are not to be extended to any type other than wide  or logical where of a fundamental  type 
2) functions requiring ordered types
3) functions that are or will potentially be extended to complex

These parts being divided if necessary to things as arithmetic trigonometry bit_wise etc.

In fact it seems to me this kind of subdivision is more sensible for users, but I am not sure.
What do you think of that ? Can you give pro and con and suggestions to improve the pattern if you think it is not stupid.
Perhaps this must wait until a new good concept hierarchy is filled into functions implementations?

**Comment by jfalcou** [comment]
complex has been moved tot heir own library solvign this particular issue
